core/upgrade/upgrades.php

```diff
@@ -660,8 +661,8 @@ public static function _v_3_0_0_move_general_settings_to_kit( $updater ) {
 
 			$meta_key = \Elementor\Core\Settings\Page\Manager::META_KEY;
 			$current_settings = get_option( '_elementor_general_settings', [] );
-			$current_settings['viewport_md'] = get_option( 'elementor_viewport_md', '' );
-			$current_settings['viewport_lg'] = get_option( 'elementor_viewport_lg', '' );
+			$current_settings[ Responsive::BREAKPOINT_OPTION_PREFIX . 'md' ] = get_option( 'elementor_viewport_md', '' );
```

Comment:
On upgrade you can't use consts that may be change, it must be hard coded the exactly string that expected in the old data.

Reply:
That's why I didn't touch the old data. Only the new settings that must be the same as the current breakpoints key

Reply:
`$current_settings` is refer to data from `_elementor_general_settings` that has the keys `viewport_md` and not something dynamic.